### ISSUE TITLE
Using ruzstd

### ISSUE THREAD

**Issue by kpcyrd** [Nullus157/async-compression#291]
hello!

I was wondering, are there any plans to support the [`ruzstd`](https://crates.io/crates/ruzstd) crate as pure Rust compression/decompression backend to avoid a dependency on libzstd.so? :) 

Thanks!

**Comment by zuisong** [comment]
https://github.com/KillingSpark/zstd-rs/releases/tag/v0.8.0

ruzstd v0.8.0 add support for compression.

**Comment by CathalMullan** [comment]
Another 'zstd in Rust' implementation was also announced recently: https://trifectatech.org/blog/announcing-zstandard-in-rust

**Comment by NobodyXu** [comment]
> Another 'zstd in Rust' implementation was also announced recently: https://trifectatech.org/blog/announcing-zstandard-in-rust


Yeah I'd prefer trifectatech, they have a good tracking record for zlib,  sudo-rs and all the other work

**Comment by paddor** [comment]
You might be interested in [zrip](https://github.com/paddor/zrip) and [lz4rip](https://github.com/paddor/lz4rip)
